src/data_collection.py

```diff
@@ -224,9 +254,8 @@ def calculate_individual_metrics(json_file_name):
             data_dict[key]["RATIO"] = average
             formats = get_file_formats(data_dict[key]["FILES"])
             data_dict[key]["FORMAT"] = formats
-        # Reformat the dictionary as a value of the key INDIVIDUAL_METRICS
-        indvividual_metrics_dict = {"INDIVIDUAL_METRICS": data_dict}
-        return indvividual_metrics_dict
+        return data_dict
+    # if RAW_DATA key was not found, empty dictionary will be returned
     return {}
```

Comment:
we should consider adding a print statement here to notify that a Dictionary was not found.

Reply:
This will be handled in the main file where the function is called